tools/gen-providers.ts

```diff
@@ -1140,6 +1140,68 @@ const launch_configuration: RecordDecl = {
   ],
 };
 
+// Minimal LaunchTemplate
+const launch_template_block_device_mapping_ebs: RecordDecl = {
+  name: 'launch_template_block_device_mapping_ebs',
+  fields: [
+    optionalField('delete_on_termination', BOOLEAN),
+    // optionalField('encrypted', BOOLEAN),
+    optionalField('iops', NUMBER),
+    // kms_key_id
+    optionalField('snapshot_id', STRING),
+    optionalField('throughput', NUMBER),
+    optionalField('volume_type', enumType(["standard", "gp2", "io1", "io2", "sc1", "st1"])),
```

Comment:
As of current hx-terrform you would want to refer only to the (old) version of AWS provider.

https://registry.terraform.io/providers/hashicorp/aws/2.70.0/docs/resources/launch_template#volume_type

(has fewer volume_type options)

Reply:
I literally copied it from https://github.com/helix-collective/hx-terraform/blob/25e0e27425455d295f4b1d6f468c590e62a892a5/tools/gen-providers.ts#L42